tech/languages/python/scipy.md

```diff
@@ -25,6 +25,8 @@ available in Fedora. Well known and widely used is
 * [SymPy](http://www.sympy.org/en/index.html), a Python library for computer algebra
   support (i.e. symbolical computation)
 * [nose](http://nose.readthedocs.io/en/latest/), a testing framework for Python.
+* [Jupyter Notebook](https://jupyter.org/), a web app that allows you to create
+and share live code, equations, visualizations and explanatory text
```

Comment:
Also, I have noticed that the last bullet point in this list had a full stop. Now it looks weird. (Sorry for not catching this earlier).